plugins/examples/sidebar/src/index.tsx

```diff
@@ -85,3 +94,28 @@ registerRoute({
     </SectionBox>
   ),
 });
+
+// The sidebar link URL is: /no-cluster-link
+registerSidebarEntry({
+  parent: null,
+  name: 'no-cluster-link',
+  label: 'No Cluster Link',
+  url: '/no-cluster-link',
+  icon: 'mdi:comment-quote',
+  useClusterURL: false, // Do not put "/c/mycluster/" in the URL
+});
+
+// This component rendered at URL: /no-cluster-link
+registerRoute({
+  path: '/no-cluster-link',
+  sidebar: null,
+  name: 'no-cluster-link',
+  exact: true,
+  useClusterURL: false, // This one does not have a "/c/mycluster/" in the URL
```

Comment:
You can test this PR is still backwards compatible with the old "noCluster: true" option.

Set "noCluster: true" here.

Reply:
@illume what's the difference b/w noCluster and useClusterURL?? Does the noCluster option also doesn't allow to put a url without "/c/mycluster" ??

Reply:
noCluster was the name of the option previously. In this PR noCluster is being deprecated.

```ts
useClusterURL: true IS noCluster: false
useClusterURL: false IS noCluster: true
```

Now registerRoute, and registerSidebarEntry both use the useClusterURL option.

Reply:
got it thanks, also For a view that is not the normal /c/mycluster should we also allow an option to not show the appbar??

Reply:
Do you need that option?

Reply:
Ashu added an issue for no-app-bar https://github.com/kinvolk/headlamp/issues/681